Scripts/Python/nb01RPSGame.py

```diff
@@ -941,17 +950,27 @@ def _HandleVariableNotify(self, events):
         for event in events:
             args[event[1]] = event[3]
 
-        # Now, let's fire it off!
-        type = args["type"]
-        del args["type"]
-        if type not in self._event_handlers:
-            PtDebugPrint("nb01RPSGame._HandleVariableNotify():\tPyEvent '{}' doesn't have a handler!".format(type))
-            return False
-        try:
-            self._event_handlers[type](**args)
-        except TypeError:
-            PtDebugPrint("nb01RPSGame._HandleVariableNotify():\tPyEvent '{}' has bad kwargs".format(type))
-        return True
+        if "YesNo" in args:
+            PtDebugPrint("YesNo is",args["YesNo"])
+            if args["YesNo"]:
+                PtDisableControlKeyEvents(self.key)
+                PtEnableMovementKeys()
+                PtAvatarExitAFK()
+                return True
+            else:
+                return False
+        elif "type" in args:
```

Comment:
```suggestion
        else:
```

Previously, the code would raise a `KeyError` if `type` is missing. We would still like for that to happen so that we can sniff out incorrect notifications.